Library/Formula/libcppa.rb

```diff
@@ -2,8 +2,8 @@ class Libcppa < Formula
   # TODO: since libcppa has been renamed to CAF, this formula should eventually
   # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.12.1.tar.gz"
-  sha1 "a8267a2f5ab4f9124ab0b10279a6600c2867243b"
+  url "https://github.com/actor-framework/actor-framework/archive/0.12.2.tar.gz"
```

Comment:
Should work, aye. If it doesn't, you should be able to do something like rewording the commit message, which will throw the SHA off as well.